### ISSUE TITLE
Black pop-ups appear on buffer save or on lsp events

### ISSUE THREAD

**Issue by nickitat** [xiyaowong/transparent.nvim#54]
Hi!
First of all thanks for the awesome plugin!

I have only one place when transparency is still missing:

<img width="418" alt="Screenshot 2023-10-16 at 12 29 25" src="https://github.com/xiyaowong/transparent.nvim/assets/5218483/ef656f3f-6dd3-46de-8c44-0a24cdafe78b">

I tried searching through highlights and manually adding all highlight groups that appear in this black rectangle to `extra_groups`, but either I failed to find the right one or it is not discoverable this way.

My setup:

``` lua
-- Transparent everything
require("transparent").setup({
  groups = {
    "Comment",
    "Conditional",
    "Constant",
    "CursorLineNr",
    "EndOfBuffer",
    "Function",
    "Identifier",
    "LineNr",
    "NonText",
    "Normal",
    "NormalNC",
    "Operator",
    "PreProc",
    "Repeat",
    "SignColumn",
    "Special",
    "Statement",
    "String",
    "Structure",
    "Todo",
    "Type",
    "Underlined",
  },
  extra_groups = {
    "NormalFloat", -- plugins which have float panel such as Lazy, Mason, LspInfo
    "FloatBorder",
    "FloatShadow",
    "FloatShadowThrough",
    --"Folded",
    "NormalContrast",
    "NvimTreeNormal", -- NvimTree
    "TreesitterContext",
    "TreesitterContextLineNumber",
    --"Visual",
    --"VisualNOS"
  },
  exclude_groups = {},
})
require('transparent').clear_prefix('lualine')
```

**Comment by xiyaowong** [comment]
You should set `:winblend` for floating windows

**Comment by nickitat** [comment]
I tried to set `:winblend = {0, 50, 100}` - in all cases this black boxes remain unchanged

**Comment by xiyaowong** [comment]
What's the plugin?

**Comment by nickitat** [comment]
unfortunately I have no idea where this messages coming from. I assume the message that shows up when I save buffer is issued by the vim itself.

**Comment by xiyaowong** [comment]
Maybe j-hui/fidget.nvim ?

**Comment by nickitat** [comment]
this is one doesn't seem to be installed

**Comment by nickitat** [comment]
> issued by the vim itself.

maybe I'm wrong about this one. on freshly installed LazyVim I don't see these messages. will look closely later

**Comment by nickitat** [comment]
it is `noice`. and apparently it supports `winblend` option, but what I got is still not quite what I wanted )

<img width="414" alt="Screenshot 2023-10-17 at 00 25 45" src="https://github.com/xiyaowong/transparent.nvim/assets/5218483/94fb87ad-271c-4424-9e3a-e888e8def366">

also as I understood it should use `NoiceMini` hl group. I tried adding it to the `transparent.nvim` config - no luck. when I search in highlights for this group it says
`NoiceMini xxx cterm= gui=links to MsgArea` - does it say anything useful to you?
